### PR TITLE
Fix bug that omits generated protobuf sources from builds

### PR DESCRIPTION
**Pull request by naddeoa**
For some reason, poetry wasn't including the generated protobuf sources
in the wheel or the source tarballs. I couldn't find anything that was
excluding them so I ended up explicitly including them in the
pyproject.toml and that fixes the issue.

### CODE REVIEW

**Review by andyndang — APPROVED** [review]
LGTM

File: Makefile [hunk 1]
```diff
@@ -33,27 +33,27 @@ help:
 
 bump-patch: ## Bump the patch version (_._.X) everywhere it appears in the project
 	@$(call i, Bumping the patch number)
-	bumpversion patch --allow-dirty
+	poetry run bumpversion patch --allow-dirty
```
Comment (andyndang) [inline Makefile:36]: https://github.com/peritus/bumpversion is not being maintained. You want `bump2version`

**Comment by lalmei** [comment]
btw notice that the coverage is no longer being computed .

**Comment by lalmei** [comment]
it is not a blocker but we should fix this soon.

**Review by andyndang — APPROVED** [review]

**Pull request merged**